snapcraft/internal/pluginhandler.py

```diff
@@ -874,3 +881,23 @@ def check_for_collisions(parts):
         # And add our files to the list
         parts_files[part.name] = {'files': part_files,
                                   'installdir': part.installdir}
+
+
+def _combine_filesets(fileset_1, fileset_2):
```

Comment:
@josepth so @kyrofa is correct that if `stage` and `prime` contradict each other a conflict error should be raised.